concepticondata/conceptlists/Miklashevsky-2018-506.tsv

```diff
@@ -0,0 +1,507 @@
+Miklashevsky-2018-506-430	430	углубление	uglublenie	cavity	Ground	concrete	10	n	1	3.32	1.43	1.27	1.37	2.54	4.32	3.8	1.85	7.67	3.96	1155	CAVE
+Miklashevsky-2018-506-431	431	угольник	ugolnik	set square	Tool_precise_grip	concrete	8	m	0	4.23	1.5	1.35	1.07	3.3	4.93	5.27	3.28	7.91	1.93		
+Miklashevsky-2018-506-432	432	удар	udar	blow	Action	abstract	4	m	2.2	3.38	4.89	1.9	1.82	5.32	5.25	5.81	3.47	4.07	5.03		
+Miklashevsky-2018-506-433	433	ударение	udarenie	emphasis	Sound	concrete	8	n	0.8	2.48	4.68	1.24	1.07	1.79	3.16	2	3.19	6.78	5		
+Miklashevsky-2018-506-434	434	удивление	udivlenie	surprise	Sense_Emotion	abstract	9	n	1.8	3.47	2.57	1.96	2.32	1.89	4.42	2.48	4.68	5.5	5.43		
+Miklashevsky-2018-506-435	435	удовольствие	udovolstvie	pleasure	Sense_Emotion	abstract	12	n	2.1	2.94	2.22	3.17	4.05	3.68	4.79	2.65	5.03	6.33	5.97		
+Miklashevsky-2018-506-436	436	укроп	ukrop	dill	Food	concrete	5	m	0.7	3.45	1.48	5.46	5.38	4.45	6.89	4.93	1.84	4.52	4.11		
+Miklashevsky-2018-506-437	437	умение	umenie	skill	Intelligence	abstract	6	n	1.6	2.53	2.41	1.4	1.57	2.21	3.9	4.74	4.59	6.78	5.66		
+Miklashevsky-2018-506-438	438	уныние	unyinie	despondency	Sense_Emotion	abstract	6	n	0.8	2.52	2.13	1.62	1.76	1.21	3.14	1.9	3.18	8.07	4.17		
+Miklashevsky-2018-506-439	439	урна	urna	urn	Object	concrete	4	f	0.9	4.34	2.24	3.72	1.34	3.69	6.33	4.48	2	6.44	4.54		
+Miklashevsky-2018-506-440	440	утка	utka	duck	Animal	concrete	4	f	1.1	3.86	4.1	2.24	4	3.59	6.85	3.59	3.13	3.07	3.14	1360	DUCK
+Miklashevsky-2018-506-441	441	ухо	uho	ear	Body_Part	concrete	3	n	2.1	3.07	6.07	1.97	2.09	4.03	6.62	3.1	3.85	2.93	4.69	1247	EAR
+Miklashevsky-2018-506-442	442	уход	uhod	care	Action	abstract	4	m	1.7	2.76	1.8	1.41	1.66	1.46	3.13	4	3.18	6.13	4.38		
+Miklashevsky-2018-506-443	443	факт	fakt	fact	Intelligence	abstract	4	m	2.2	2.63	1.64	1.55	1.32	1.75	3.04	1.42	3.65	7.6	5.73		
+Miklashevsky-2018-506-444	444	фактор	faktor	factor	Intelligence	abstract	6	m	2	1.79	1.41	1.28	1.41	1.34	2.41	1.76	3.84	9.44	4.71		
+Miklashevsky-2018-506-445	445	фантазия	fantaziya	fantasy	Intelligence	abstract	8	f	1.5	3.52	1.93	1.77	2.19	1.62	3.34	1.9	5.67	6.4	5.31		
+Miklashevsky-2018-506-446	446	фейерверк	feyerverk	fireworks	Space	concrete	9	m	0.8	5.72	5.14	3.14	1.55	2.1	6.93	4.97	5.15	6.3	2.93		
+Miklashevsky-2018-506-447	447	флаг	flag	flag	Object	concrete	4	m	1.5	4.45	2.5	1.55	1.62	3.21	6.63	3.5	5.12	4.87	4.2	2623	FLAG
+Miklashevsky-2018-506-448	448	фламинго	flamingo	flamingo	Animal	concrete	8	m	0.2	4.24	2.1	2.48	1.45	2.88	6.53	1.97	2.67	6.23	2.13	3722	FLAMINGO
+Miklashevsky-2018-506-449	449	фломастер	flomaster	soft-tip pen	Tool_precise_grip	concrete	9	m	0.5	5.1	2.39	3.27	2.64	4.74	6.59	6.59	2.63	3.38	4.07		
+Miklashevsky-2018-506-450	450	фонарь	fonar	lantern	Object	concrete	6	m	1.5	5.84	1.93	1.31	1.36	3	6.54	4.9	4.03	4.33	4.67		
+Miklashevsky-2018-506-451	451	форма	forma	form	Sense_Phys	abstract	5	f	2.5	4.59	1.55	1.52	1.64	3.31	3.96	3.43	3.61	5.33	4.41		
```

Comment:
Yes, this is a good idea. It is always disappointing to see so many nouns which are concrete enough and few map to concepticon, since they have a different spirit than the traditional basic vocabulary.